nodestream/project/project.py

```diff
@@ -11,14 +11,14 @@
 )
 from ..pipeline import Step
 from ..pluggable import Pluggable
+from ..project.plugin import PluginConfiguration
 from ..schema.schema import (
     AggregatedIntrospectiveIngestionComponent,
     GraphSchema,
     IntrospectiveIngestionComponent,
 )
 from .pipeline_definition import PipelineDefinition
 from .pipeline_scope import PipelineScope
-from .plugin import PluginConfiguration
 from .run_request import RunRequest
```

Comment:
I think this should be `from .plugin import PluginConfiguration` for consistency.